controllers/gatekeeper_controller.go

```diff
@@ -831,6 +849,21 @@ func setEmitEvents(obj *unstructured.Unstructured, argName string, emitEvents *o
 	return nil
 }
 
+func setEventsInvolvedNamespace(obj *unstructured.Unstructured,
```

Comment:
Do you have a test for this?

Reply:
Yes upstream gatekeeper side has this

Reply:
I meant the operator setting the value should have a test. See some of the other functions in that file that are similar for examples of how they are tested.

Reply:
@yiraeChristineKim , this is still unaddressed.